src/routes/prescricao/get.js

```diff
@@ -9,13 +7,9 @@ module.exports = (Prescricao, Usuario) => (req, res) => {
     if (!prescricao) {
       return res.render('pages/error')
     }
-    let tipoDoUsuario = req.user.tipo
-    if (tipoDoUsuario === 'medica') {
-      res.render('pages/prescricao', { prescricao })
-    }
-
-    if (tipoDoUsuario === 'farmaceutica') {
-      res.render('pages/editarPrescricaoFarmaceutica')
-    }
+    res.render('pages/prescricao', {
+      prescricao,
+      tipoUsuario: req.user.tipo
```

Comment:
To vendo que todas/ou quase todas nossas rotas tem esse tipoUsuário. A gente poderia colocar num `local` dentro do nosso middleware de usuário pra remover essa duplicação. O que vocês acham?